Source/bmalloc/libpas/src/libpas/pas_utils.h

```diff
@@ -53,8 +53,6 @@ PAS_IGNORE_CLANG_WARNINGS_BEGIN("qualifier-requires-header")
 #define PAS_BEGIN_EXTERN_C __PAS_BEGIN_EXTERN_C
 #define PAS_END_EXTERN_C __PAS_END_EXTERN_C
 
-PAS_BEGIN_EXTERN_C;
```

Comment:
why is this change needed in `bmalloc`?

Reply:
IIRC some transitive include from pas_utils_additions.h has a #include <> inside extern "C" which the swift compiler does not like.